assets/dev/scss/frontend/general/_global.scss

```diff
@@ -11,3 +11,11 @@
 	margin: 0;
 	line-height: 1;
 }
+
+.e-youtube-base {
```

Comment:
@rami-elementor @Nevoss Where do you think I can put this CSS rule?

Reply:
Remove file